target/linux/generic/pending-6.1/816-1-dt-bindings-arm-add-cznic-turris-omnia-mcu-binding.patch

```diff
@@ -0,0 +1,99 @@
+From 4ad2d5a8c7ab06dd0ed263510800a4e86cdd5e54 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Marek=20Beh=C3=BAn?= <kabel@kernel.org>
+Date: Tue, 22 Aug 2023 13:26:38 +0200
+Subject: [PATCH v4 1/7] dt-bindings: arm: add cznic,turris-omnia-mcu binding
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Add binding for cznic,turris-omnia-mcu, the device-tree node
+representing the system-controller features provided by the MCU on the
+Turris Omnia router.
+
+Signed-off-by: Marek Behún <kabel@kernel.org>
+---
+ .../bindings/arm/cznic,turris-omnia-mcu.yaml  | 67 +++++++++++++++++++
+ MAINTAINERS                                   |  1 +
+ 2 files changed, 68 insertions(+)
+ create mode 100644 Documentation/devicetree/bindings/arm/cznic,turris-omnia-mcu.yaml
+
+--- /dev/null
++++ b/Documentation/devicetree/bindings/arm/cznic,turris-omnia-mcu.yaml
```

Comment:
Do you plan to use this driver on other targets too? If not the patches should go into target/linux/mvebu/patches-6.1. 
If you keep it here please add the refreshing of the other patches also into this patch which broke them.

Reply:
OK, I'll move it to mvebu.